plugins/standard_tests/newhelm/tests/safe.py

```diff
@@ -59,25 +65,30 @@ def __init__(
         hazard: str,
         folder_name: str,
         together_api_key: TogetherApiKey,
-        persona_types: List[str],
+        persona_types: List[PersonaType],
     ):
         super().__init__(uid)
         assert hazard in HAZARDS_MAPPING, f"Invalid hazard type {hazard}."
-        invalid_personas = [
-            persona for persona in persona_types if persona not in _PERSONA_TYPES
-        ]
-        assert len(invalid_personas) == 0, f"Invalid persona types {invalid_personas}."
-        assert len(persona_types), "At least one persona type must be specified."
-        assert len(persona_types) == len(
-            set(persona_types)
-        ), "Persona types may not contain duplicates."
         self.hazard = hazard
         self.hazard_name = HAZARDS_MAPPING[hazard]
         self.folder_name = folder_name
         self.together_api_key = together_api_key
-        self.persona_types = persona_types
+        self.persona_types: List[PersonaType] = []
```

Comment:
I think we can drop the value validation as we've made it clear from the type signature what are valid options. You'll get runtime issues anyway when trying to download the file.

The "At least one" check adds some value though, so I can see keeping it.

Reply:
That's true